internal/db/rules.go

```diff
@@ -1,175 +1,79 @@
 package db
 
 import (
-	"fmt"
-
-	"log"
-	"strconv"
-
-	"encoding/json"
-
+	"database/sql"
+	"github.com/pkg/errors"
 	// Blank import required for mysql driver
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/jimmyjames85/bouncecm/internal/models"
 )
 
 // ListRules - Function to pull all rules from db
-func (c *Client) ListRules() (*models.RulesObject, error) {
+func (c *Client) GetAllRules() (*models.RulesObject, error) {
 	rules := []models.BounceRule{}
-
 	rows, err := c.Conn.Query("SELECT * FROM bounce_rule")
 
 	if err != nil {
-		log.Println(err)
-		fmt.Println("FAILEDHERE?")
-
-		return nil, err;
-
+		return nil, errors.Wrap(err, "GetAllRules Query")
 	}
 
+	defer rows.Close()
+
 	for rows.Next() {
 		br := models.BounceRule{}
-		var description *string
-		err := rows.Scan(&br.ID, &br.ResponseCode, &br.EnhancedCode, &br.Regex, &br.Priority, &description , &br.BounceAction)
-		
-		// case when description is null becuase it can be null according to droprules.sql
-		if description == nil {
-			br.Description = ""
-		} else {
-			br.Description = *description
-		}
 
+		var description sql.NullString
+		err := rows.Scan(&br.ID, &br.ResponseCode, &br.EnhancedCode, &br.Regex, &br.Priority, &description, &br.BounceAction)
 
+		if description.Valid {
+			br.Description = description.String
+		} else {
+			br.Description = ""
```

Comment:
This is else block isn't necessary as `br.Description` will be populated with it's zero value which is empty string.  See: https://tour.golang.org/basics/12